docs/node-operators/systemd-service.md

```diff
@@ -4,6 +4,12 @@ The `runner.sh` script automatically downloads and installs the latest version o
 
 For Testnet 7.3 and greater, the configuration file needs to be customized with the public IP and port of the node, which must be set at the `public_addr` field in `witnet.toml` .
 
+* Install latest version of ca-certificates to download safely from GitHub:
+
+```sh
+sudo apt install ca-certificates
```

Comment:
Note that some people may already have the `ca-certificates` pre-installed, but the certificates themselves may be out of date and could need also this:
```console
sudo update-ca-certificates
```

Reply:
on my Ubuntu Bionic node installed by the same script it worked without having ca-certificates package installed

Reply:
I think it could make sense to mention it as a side note (just in case)

Reply:
yes I agree